.travis.yml

```diff
@@ -32,7 +33,6 @@ before_install:
     - source venv/bin/activate
     - python --version # just to check
     - retry pip install nose # always
-    - sudo apt-get install libblas-dev liblapack-dev libatlas3gf-base
```

Comment:
I think you need to add something like:

```
cache: pip
addons:
  apt:
    packages:
    - ibblas-dev
    - liblapack-dev
    - libatlas3gf-base
```

https://github.com/scikit-image/scikit-image/pull/1607